pkg/backend/remote/remote.go

```diff
@@ -211,16 +211,16 @@ func (r *Remote) SectorSize() (int64, error) {
 	return replicaInfo.SectorSize, nil
 }
 
-func (r *Remote) RemainSnapshots() (int, error) {
+func (r *Remote) GetSnapshotCountAndSizeUsage() (int, int64, error) {
```

Comment:
nit: for multiple returned values, suggest having named variables.

Reply:
I updated the one in `pkg/controller/replicator.go`. We don't have calculation in remote.go. We just return result from another function, so I don't update it.